content/millennium/r4/individuals/related-person.md

```diff
@@ -109,13 +125,25 @@ List an individual RelatedPerson by its id:
 <%= headers status: 200 %>
 <%= json(:r4_relatedperson_entry) %>
 
+### Example - Encounter level
+
+#### Request
+
+    GET https://fhir-ehr-latest.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/RelatedPerson/E-12457994-97697434
```

Comment:
Let's change this to fhir-open.cerner.com to be consistent with the other request urls.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/526/commits/b3d1178567f0c571011e9524525c7a7d6e69fd6a#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR133

Reply:
Don't need the -latest

Reply:
Updated: https://github.com/cerner/fhir.cerner.com/pull/526/commits/756e1342235d0174072f3a2e2498873c1cf83cd3#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR133